zulipterminal/helper.py

```diff
@@ -493,3 +493,95 @@ def notify(title: str, html_text: str) -> None:
     if command:
         res = subprocess.run(shlex.split(command), stdout=subprocess.DEVNULL,
                              stderr=subprocess.STDOUT)
+
+
+def parse_html_table(table_element: Any) -> Tuple[List[str], List[List[str]]]:
+    """
+    Parses an HTML table to extract cell items and column alignments.
+    """
+    headers = table_element.thead.tr.find_all('th')
+    rows = table_element.tbody.find_all('tr')
+    alignments = []
```

Comment:
This is on a column-basis, right? So maybe `column_alignments`?

Reply:
Yes, `column_alignments` is a better choice.

Reply:
Yes, it on a column-basis, `column_alignments` will be perfect.

Reply:
Yes, it is on a column-basis, column_alignments will be perfect.

Reply:
Yes, it is on a column-basis, `column_alignments` will be perfect.